apache-maven/src/assembly/maven/conf/maven.properties

```diff
@@ -0,0 +1,41 @@
+#
+# Maven user properties
+#
+
+maven.installation.conf = ${maven.home}/conf
+maven.user.conf         = ${user.home}/.m2
+maven.project.conf      = ${session.rootDirectory}/.mvn
+
+# Space separated list of files to load.
+# If the first character is a question mark, the load will silently fail if the file does not exist.
+${includes} = ?${maven.user.conf}/maven.properties, \
```

Comment:
There is a logical question here to what extend should the question mark apply?  Until the last "}"? From a logical PoV it should not include `/maven.properties.

Reply:
I thinking whether there are already used syntaxes people might know already.